sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/page_object/android/ApiDemosListViewScreenChaided.java

```diff
@@ -33,48 +29,48 @@
 	 * It was done so just for obviousness
 	 */
 
-	@AndroidFindBys({
+	@AndroidFindBySet({
 		@AndroidFindBy(uiAutomator = "new UiSelector().resourceId(\"android:id/list\")"), //the searching

```

Comment:
@SrinivasanTarget there is other example for that 

https://github.com/saikrishna321/sample-code/blob/master/sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/page_object/android/ApiDemosListViewScreenByAllPossible.java#L36-L39

Thought we could one with Set